docs/recipes_api.rst

```diff
@@ -26,6 +26,11 @@ Robotics Kit: Alex Robot - Line Detection
 
 .. literalinclude:: ../examples/robotics/alex_line_detect.py
 
+Displaying camera stream in pi-top [4]'s miniscreen
+------------------------------------------------------------
+
+.. literalinclude:: ../examples/camera/camera_display_in_miniscreen.py
```

Comment:
How do we handle the examples that combine things together? Should there be an `examples/recipes` folder?